LICENCE

```diff
@@ -6,24 +6,48 @@ in the prover_snapshots directory (which include copies of their
 licences), is subject to the BSD two-clause licence below.
 
 Copyright (c) 2017-2023
-  Prashanth Mundkur
-  Rishiyur S. Nikhil and Bluespec, Inc.
-  Jon French
-  Brian Campbell
-  Robert Norton-Wright
-  Alasdair Armstrong
-  Thomas Bauereiss
-  Shaked Flur
-  Christopher Pulte
-  Peter Sewell
-  Alexander Richardson
-  Hesham Almatary
-  Jessica Clarke
-  Microsoft, for contributions by Robert Norton-Wright and Nathaniel Wesley Filardo
-  Peter Rugg
-  Aril Computer Corp., for contributions by Scott Johnson
-  Philipp Tomsich
-  VRULL GmbH, for contributions by its employees
+   ahadali5000
+   Alasdair Armstrong
+   Alexander Richardson
+   Aril Computer Corp., for contributions by Scott Johnson
+   Ben Marshall
+   Bicheng Yang
+   Bilal Sakhawat
+   Brian Campbell
+   Chris Casinghino
+   Christopher Pulte
+   Codasip, for contributions by Tim Hutt
+   dylux
+   eroom1966
+   Hesham Almatary
+   James Clarke
+   Jan Henrik Weinstock
+   Jessica Clarke
+   Jon French
+   Martin Berger
+   Michael Sammler
+   Microsoft, for contributions by Robert Norton-Wright and Nathaniel Wesley Filardo
+   Muhammad Bilal Sakhawat
+   Nathaniel Wesley Filardo
+   Paul A. Clarke
+   Peter Rugg
+   Peter Sewell
+   Philipp Tomsich
+   Prashanth Mundkur
+   Rafael Sene
+   Rishiyur S. Nikhil (Bluespec, Inc.)
+   Robert Norton-Wright
+   Scott Johnson (Aril Inc.)
+   Shaked Flur
+   Thibaut Pérami
+   Thomas Bauereiss
+   VRULL GmbH, for contributions by its employees
+   William McSpaddden
+   Xinlai Wan
```

Comment:
```suggestion
   Alasdair Armstrong
   Alexander Richardson
   Aril Computer Corp., for contributions by Scott Johnson
   Ben Marshall
   Bicheng Yang
   Bilal Sakhawat
   Brian Campbell
   Chris Casinghino
   Christopher Pulte
   Codasip, for contributions by Tim Hutt
   Google LLC, for contributions by its employees
   Hesham Almatary
   Jan Henrik Weinstock
   Jessica Clarke
   Jon French
   Martin Berger
   Michael Sammler
   Microsoft, for contributions by Robert Norton-Wright and Nathaniel Wesley Filardo
   Muhammad Bilal Sakhawat
   Nathaniel Wesley Filardo
   Paul A. Clarke
   Peter Rugg
   Peter Sewell
   Philipp Tomsich
   Prashanth Mundkur
   Rafael Sene
   Rishiyur S. Nikhil (Bluespec, Inc.)
   Robert Norton-Wright
   Scott Johnson (Aril Inc.)
   Shaked Flur
   Thibaut Pérami
   Thomas Bauereiss
   VRULL GmbH, for contributions by its employees
   William McSpaddden
   Xinlai Wan
```

Maybe we should drop the GitHub IDs without names her and just leave companies and names? Probably also need to add Google here for my contributions after joining.

Reply:
Happy to keep them.